app/src/main/java/fr/free/nrw/commons/delete/DeleteTask.java

```diff
@@ -173,6 +173,7 @@ protected void onPostExecute(Boolean result) {
             message = "Could not request deletion.";
         }
 
+
         notificationBuilder.setDefaults(DEFAULT_ALL)
```

Comment:
Please remove this one too. After sending a PR you can click to Files changed tab and see which changes you made. For your task, changing DeleteTask.java is not necessary so don't change it. Always make sure that you edited just needed lines for your task.